src/components/SurveyQuestion/SurveyQuestion.module.css

```diff
@@ -0,0 +1,7 @@
+.ratingQuestion .activeAnswer {
+  @apply opacity-100;
+}
+
+.ratingQuestion .inactiveAnswer {
+  @apply opacity-50;
+}
```

Comment:
As we have a function dedicated to `answerClass`, should we directly use the Tailwind class inside this answerClass function?

Reply:
Updated in [b076636](https://github.com/hanam1ni/react-surveyor/pull/79/commits/b076636da5d3d95c56a7ea4528ad9ad708bceb65)